src/index.ts

```diff
@@ -14,129 +12,24 @@ const argv = minimist(process.argv.slice(2), {
 });
 
 async function app(argv: any) {
-  if (!argv.accountId) {
-    throw new Error(`No [--accountId] set, this is required`);
-  }
-  if (!argv.packageName) {
-    throw new Error(`No [--packageName] set, this is required`);
-  }
-
-  const verbose = argv.verbose || false;
-  if (!argv.verbose) {
-    logger.log(`[--verbose] is not set, defaulting to [${verbose}]`);
-  }
-  (process as any).verbose = verbose;
-
-  const parallel = argv.parallel || 1;
-  if (!argv.parallel) {
-    logger.log(`[--parallel] is not set, defaulting to [${parallel}]`);
-  }
-
-  const daysToScrape = argv.days || 7;
-  if (!argv.days) {
-    logger.log(`[--days] is not set, defaulting to [${daysToScrape}]`);
-  }
-  if (![1, 7, 14, 30, 60].includes(daysToScrape)) {
-    throw new Error(`[--days=${argv.days}] is invalid, please supply one of: [1, 7, 14, 30, 60]`);
-  }
-
-  const format = argv.format || 'json';
-  if (!argv.format) {
-    logger.log(
-      `[--format] is not set, defaulting to [${format}] (options: json|csv)`
-    );
-  }
-  if (format !== 'json') {
-    throw new Error(`Currently only supports [--format=json]`);
-  }
-
-  let outputDir;
-  if (!argv.outDir) {
-    outputDir = process.cwd();
-    logger.log(`[--outDir] is not set, using [${outputDir}]`);
+  let mode: 'crashes' | 'overview' = 'crashes';
```

Comment:
Would it be practical to use the contents of validModes = ['crashes', 'overview']; here instead of typing the values several times?

Reply:
One is defining types, and one is defining values. We could probably improve this to some degree, but not a lot